tests/test_cmfd_linear_source/test_cmfd_linear_source.py

```diff
@@ -28,6 +28,7 @@ def _create_geometry(self):
         cmfd.setLatticeStructure(17,17)
         cmfd.setGroupStructure([[1,2,3], [4,5,6,7]])
         cmfd.setKNearest(3)
+        cmfd.setCentroidUpdateOn(False)
```

Comment:
This disables nearest neighbors, preventing success of the test.
Without this line it passes on my machine

Reply:
I will try it again.